### PR TITLE
add rings and bracelets

### PR DESCRIPTION
**Pull request by curstwist**
#### Summary
```SUMMARY: Content "Adds gem stone rings and braclets."```

#### Purpose of change
continue jewelry expansion.

#### Describe the solution
add more.

#### Testing
loaded world, spawned items, checked looks_like and did spawn checks with debug and in game spawns for item_groups.

### CODE REVIEW

**Comment by Jerimee** [comment]
At the risk of being trifling, is it desirable that aquamarine_gold_bracelet be given the same probability as ruby_ or emerald_? My understanding is that some precious stones are rarer than others.

**Comment by curstwist** [comment]
> 
> 
> At the risk of being trifling, is it desirable that aquamarine_gold_bracelet be given the same probability as ruby_ or emerald_? My understanding is that some precious stones are rarer than others.

I'm not going to get into probabilities for spawns based on gem type.  If someone else wants to get that detailed once they are all in, they are welcome to pursue it.

**Comment by I-am-Erk** [comment]
Can you please change these to use the new format for names, eg 
```
    "name": { "str": "soap dish", "str_pl": "soap dishes" },
```

**Comment by curstwist** [comment]
> 
> 
> Can you please change these to use the new format for names, eg
> 
> ```
>     "name": { "str": "soap dish", "str_pl": "soap dishes" },
> ```

fixed, including the original PR of jewelry (earrings & dental grills)

File: data/json/items/armor/jewelry.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
   {
     "id": "diamond_dental_grill",
     "type": "ARMOR",
-    "name": "diamond dental grill",
+    "name": { "str": "diamond dental grill", "str_pl": "diamond dental grills" },
```
Comment (anothersimulacrum) [inline data/json/items/armor/jewelry.json]: If the `str_pl` just adds an `s` at the end, you don't need to include it.

**Pull request merged**